boomer/src/main.rs

```diff
@@ -36,17 +36,24 @@ struct CliArgs {
     )]
     device: PathBuf,
 
+    #[arg(short = 'C', long, help = "Connector name, for example: HDMI-A-1")]
+    connector: Option<String>,
+
     #[arg(short, long, action = clap::ArgAction::Count)]
     verbose: u8,
 }
 
-fn find_connector(device: &Device) -> Option<Rc<Connector>> {
+fn find_connector(device: &Device, connector_name: &Option<String>) -> Option<Rc<Connector>> {
```

Comment:
`Option<&str>` ?

Reply:
True, my Rust is... rusted. You'll need some patience with this kind of mistakes.